### PR TITLE
Adding Grid And Rating Component

### PR DESCRIPTION
**Pull request by MayankKumar10**
Adding Grid And Rating Component

### CODE REVIEW

File: src/styles/Modal.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 100px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
```
Comment (ShekharDhangar) [inline src/styles/Modal.css:12]: use this as variables like bg-opacity, and try to use as many helper classes

File: src/styles/Rating.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.rate {
+  float: left;
```
Comment (ShekharDhangar) [inline src/styles/Rating.css:2]: any reason to use float, instead of flex

**Review by ShekharDhangar — APPROVED** [review]
Create helper class for general css like flex,overflow, fixed, absolute, etc and use them everywhere

File: src/styles/Rating.css (hunk 2 above)
Comment (MayankKumar10) [reply src/styles/Rating.css:2]: I used Grid in this

**Pull request merged**